src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -427,6 +460,7 @@ static void Main()
             Assert.Null(model.GetTypeInfo(def).Type);
             Assert.Null(model.GetSymbolInfo(def).Symbol);
             Assert.Null(model.GetDeclaredSymbol(def));
+            Assert.Equal(0, model.GetConstantValue(def).Value);
```

Comment:
It doesn't look like they agree.

---
In reply to: [102340554](https://github.com/dotnet/roslyn/pull/17205#discussion_r102340554) [](ancestors = 102340554)